.prettierrc

```diff
@@ -1 +1 @@
-{}
\ No newline at end of file
+{}
```

Comment:
can add this file to gitignore

Reply:
It is the config file for prettier and i think it is not need to be added to gitignore.